tensorly/tenalg/__init__.py

```diff
@@ -3,7 +3,8 @@
 operations such as khatri-rao or kronecker product, n-mode product, etc.
 """ 
 
-from .n_mode_product import mode_dot, multi_mode_dot
+from .n_mode_product import (mode_dot, multi_mode_dot,
+                             unfolding_dot_khatri_rao)
```

Comment:
Maybe it should also be added to the sparse backend, since it is known to be sparse safe.

Reply:
Yes, forgot about this.